app/static/pages/home/tabs_template/volunteer_tab.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<div class="oppia-foundation-white-background"
+     layout="column"
+     layout-align="center center">
+  <md-content class="oppia-foundation-horizontal-padding
+                     oppia-foundation-white-background
+                     home-page-donate-tab-content"
+              layout="row"
+              layout-align="center">
+    <p class="oppia-foundation-no-margin-top
+              home-page-action-result-text"
+       flex="50">
+       We have a broad community of volunteers who play critical
```

Comment:
I feel like I've read lots of this text before. Can we parameterize it so that it is only stated once (possibly in JS)? In general try not to duplicate stuff.

Reply:
I put in a scope variable to manage these paragraphs.